mod_reforged/hooks/skills/skill_container.nut

```diff
@@ -87,6 +87,11 @@
 	q.onCostsPreview = @(__original) { function onCostsPreview( _costsPreview )
 	{
 		__original(_costsPreview);
-		_costsPreview.fatiguePreview += this.getActor().RF_getZOCEvasionFatigue();
+
+		local actor = this.getActor();
+		if (actor.isPreviewing() && actor.getPreviewSkill() == null)
```

Comment:
Instead of skill it would be better to check for `actor.getPreviewMovement() != null` to be clear about the intent here.